crates/router/src/routes/payments.rs

```diff
@@ -270,8 +358,24 @@ pub async fn payments_connector_session(
     .await
 }
 
+/// Payments - Redirect response
+///
+/// To get the payment response for redirect flows
+#[utoipa::path(
+    post,
+    path = "/payments/{payment_id}/{merchant_id}/response/{connector}",
+    params(
+        ("payment_id" = String, Path, description = "The identifier for payment"),
+        ("merchant_id" = String, Path, description = "The identifier for merchant"),
+        ("connector" = String, Path, description = "The name of the connector")
+    ),
+    responses(
+        (status = 200, description = "Recieved payment redirect response"),
+        (status = 400, description = "Missing Mandatory fields")
```

Comment:
```suggestion
        (status = 200, description = "Received payment redirect response"),
        (status = 400, description = "Missing mandatory fields")
```

Is this supposed to be a 200 status code or 30X?